games-engines/openmw/openmw-0.44.0.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils readme.gentoo-r1
+
+DESCRIPTION="Open source reimplementation of TES III: Morrowind"
+HOMEPAGE="https://openmw.org/"
+SRC_URI="https://github.com/OpenMW/openmw/archive/${P}.tar.gz"
+
+LICENSE="GPL-3 MIT BitstreamVera ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc devtools +qt5"
+
+RDEPEND="
+	dev-games/mygui
+	>=dev-games/openscenegraph-3.5.5[ffmpeg,jpeg,png,sdl,svg,truetype,zlib]
```

Comment:
`dev-games/openscenegraph-3.5.5` is the only version available in the tree and will be automatically installed. please update this line to `dev-games/openscenegraph`.

Reply:
Really?? An older version might already be installed, it won't automatically bump just because that version is gone. I try not to overdo version constraints but if it's known that this version is required then it should stay.

Reply:
it's just a matter of staying consistent and up to date with the tree.